### DISCUSSION TITLE
Multiple annotators working simultaneously

### DISCUSSION THREAD

**Discussion by lizfischer** [doccano/doccano#1773] [Q&A]
Hi y'all! I'm a little confused by the collaboration features of doccano. I set up a test project to play around in & have added multiple users to it (one admin, two annotators), and have "share annotations across all users" checked, but I don't see how to make it work in practice as a collaborative tool. As far as I can tell, you can't really "assign" sections of the data to individuals, so if multiple people are working at the same time, how do you not end up with duplicated work (everyone annotating every document)? I'm not sure if these features are just not very robust yet, or if I'm missing something in configuration!

Related, is there any way to see which pieces of the dataset have been labeled, which are labeled but need approval, and which are totally done? Or to see which piece of data a comment belongs to from the comments tab? I feel like I can't tell what piece of data "need attention" right now (whether that's needing attention from an annotator, an annotation approver, etc)

**Comment by david-engelmann** [comment]
You could tell your annotators to work the sets in opposite directions one going from the top to bottom, one going from the bottom to top. If annotations are shared, they should be able to tell when they've meet in the middle.

As far as, evaluating the completeness of datasets, I found it easiest to use the doccano-client library. Pull the examples by project and check the conditions you mentioned.
